02_getting_started.asciidoc

```diff
@@ -9,6 +9,24 @@ The Lightning Network is accessed via software applications that can speak the L
 
 Users have the highest degree of control by running their own Bitcoin node and Lightning node. However, Lightning nodes can also use a lightweight Bitcoin client (commonly referred to as Simplified Payment Verification (SPV)) to partially validate the correctness of their blockchain.
 
+=== Lightning Explorer
+
+Lightning network explorer is an useful tool to show the statistics of nodes, channnel and network capacity.
+
+The below is an unexhausted list:
```

Comment:
maybe instead of third party services that might potentially invade privacy we should link to open source software that can be run as an explorer? for example https://github.com/chemicstry/recksplorer

Reply:
unexhausted -> inexhaustive